app/models.py

```diff
@@ -160,7 +161,8 @@ def is_active(self):
 
 
 class Token(Base):
-    id = sa.Column(sa.String, nullable=False, primary_key=True)
+    id = sa.Column(sa.String, default=str(uuid.uuid4()), nullable=False,
```

Comment:
This should be `default=lambda: str(uuid.uuid4())`; right now you'll get the same UUID every time.